test/e2e/match_test.go

```diff
@@ -53,7 +49,7 @@ func TestFetchMatches(t *testing.T) {
 		{
 			"expects unavailable code since there is no mmf being hosted with given function config",
 			&pb.FunctionConfig{
-				Host: mmfTc.GetHostname(),
+				Host: "om-matchfunction",
```

Comment:
Why is this hard coded now? The `GetHostname` may return "localhost" for minimatch.

Reply:
Cuz OMCluster does not expose mmf text context. In addition, this test cases expect an invalid FunctionConfig, which is okay even if we pass in a random string.